manifests/kvm-ds.yml

```diff
@@ -0,0 +1,29 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    name: device-plugin-kvm
+  name: device-plugin-kvm
+spec:
+  selector:
+    matchLabels:
+      name: device-plugin-kvm
+  template:
+    metadata:
+      labels:
+        name: device-plugin-kvm
+    spec:
+      containers:
+      - name: device-plugin-kvm
+        image: quay.io/kubevirt/device-plugin-kvm
+        #args: ["-v3", "-logtostderr"]
```

Comment:
Why is this commented out? The logging shouldn't be too aggressive and is essential for debugging.

Reply:
Ah, I thought in normal opertaion it's not needed, but handy if you want to debug.